ansible_base/lib/utils/encryption.py

```diff
@@ -84,27 +85,18 @@ def is_encrypted_string(self, value: Any) -> Tuple[bool, Optional[str], Optional
         if not isinstance(value, str):
             value = str(value)
 
+    def encrypt_string(self, value: Any) -> str:
+        # Its possible for a serializer to accept a number for a CharField (like 5). In the serializer its "5" but when we get here it might be 5
+        if type(value) is not str:
+            value = json.dumps(value)
+
```

Comment:
This got merged into the middle of the `is_encrypted_function` it needs to be moved below the `return False, None, None`.